### PR TITLE
[terra-arrange] add reflow support for 400% zoom

### PR DESCRIPTION
**Pull request by gt106551**
###  Terra arrange support resizing to 400%
https://github.com/cerner/terra-core/assets/126649018/10e80b85-b6b9-4ab3-8422-2433f8ec5a70
### Terra arrange viewport is resized to 320x256 px without loss of informatio or funtion
https://github.com/cerner/terra-core/assets/126649018/628a35b2-1aee-4a0b-8c98-b3958248e36f

 **Summary**
 Please ensure that to test this with terra arrange:-
1.Ensure that terra arrange support altering multiple CSS attributes without loss of function or information.
2.Ensure that terra arrange support resizing to 400%.
3.Ensure that terra arrange support reflow when the viewport is resized to 320x256 px without loss of information or functio

n**_**Solution**_**
I have made changes in Arrange.module.scss

    max-width: 100%;
    overflow: auto;
    max-height: 100%;
    
   [UXPLATFORM-9326]

Thank you for contributing to Terra.
**@cerner/terra**

### CODE REVIEW

**Comment by sdadn** [comment]
How was this tested?

**Comment by mjpalazzo** [comment]
Where did the 400% requirement come from? Per WCAG text should be able to size to 200%. See WCAG 1.4.4 Resize Text Level AA: [https://www.w3.org/WAI/WCAG21/Techniques/general/G178.html]

**Comment by gt106551** [comment]
> Where did the 400% requirement come from? Per WCAG text should be able to size to 200%. See WCAG 1.4.4 Resize Text Level AA: [https://www.w3.org/WAI/WCAG21/Techniques/general/G178.html]



I got a comment on jira  from UX team regarding this changes.
[https://jira2.cerner.com/browse/UXPLATFORM-8662](url)

**Comment by rahulkumar8cs** [comment]
@mjpalazzo  400% browser Zoom needs to be validated in order to meet the WCAG 2.1 [1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) requirement.

**Comment by mjpalazzo** [comment]
Thanks for your response @rahulkumar8cs. I think the requirement in the Summary is not stated correctly. The intent of WCAG 1.4.10 is "When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary." That is different from what the summary says, "Ensure that Terra Arrange supports resizing to 400%" I think the requirement is that Terra Arrange reflows to one column when zoomed to 400%. The goal is to avoid the need for the user to scroll.

File: packages/terra-arrange/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed issue terra arrange support resizing to 400% and viewport is resized to 320x256 px without loss of information or function.
```
Comment (sdadn) [inline packages/terra-arrange/CHANGELOG.md:6]:
```suggestion
* Fixed
  * Fixed reflow issue when zoomed to 400% and the viewport is resized to 320x256 px.
```

**Review by sdadn — CHANGES_REQUESTED** [review]

File: packages/terra-arrange/CHANGELOG.md (hunk 1 above)
Comment (gt106551) [reply packages/terra-arrange/CHANGELOG.md]: i have made changes as per suggestion.

**Review by supreethmr — APPROVED** [review]

**Review by saket2403 — APPROVED** [review]

**Comment by ShettyAkarsh** [comment]
@gt106551  is this changes tested for different zoom size and viewpors ?

**Comment by gt106551** [comment]
> @gt106551 is this changes tested for different zoom size and viewpors ?

@ShettyAkarsh yes i have tested with different zoom size and viewport.

**Pull request merged**